showcase-docs/getting-started.md

```diff
@@ -2,6 +2,152 @@
+
+This ensures a thorough understanding of user interactions with the application while maintaining user anonymity and privacy.
+
+The data will be used only for internal analysis and product improvement.
+
+If you wish to enable telemetry data collection, follow the steps below.
+
+### Enable Telemetry
+
+To turn on the telemetry you need to enable Segment provider plugin.
+
+#### When using Helm Chart
+
+Modify Helm values and add the following configuration:
+
+```yaml
+global:
+  dynamic:
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: false
+```
+
+#### When using RHDH Operator
+
+If you already have `dynamic-plugins-rhdh` ConfigMap as described in [Configuring dynamic plugins with the Red Hat Developer Hub Operator](https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.1/html-single/administration_guide_for_red_hat_developer_hub/index#configuring-dynamic-plugins-with-the-red-hat-developer-hub-operator) you just need to add the Segment plugin to the list of plugins. Refer to `ConfgiMap` example below.
+
+If you don't have the `dynamic-plugins-rhdh` ConfigMap, you can create it with the following content:
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+data:
+  dynamic-plugins.yaml: |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: false
+```
+
+Don't forget to add `dynamicPluginsConfigMapName` referencing the above `ConfigMap` to your `Backstage` resource:
+
+```yaml
+spec:
+  application:
+    dynamicPluginsConfigMapName: dynamic-plugins-rhdh
+```
+
+### Customizing Telemetry Destination
+
+By default, the Segment plugin is configured to send data to Red Hat. If you wish to change the destination, you can do so by setting the `SEGMENT_WRITE_KEY` environment variable to the desired Segment write key.
+
+#### When using Helm Chart
+
+Add the following configuration to your
+
+```yaml
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: SEGMENT_WRITE_KEY
+        value: <segment_key>
+```
+
+#### When using RHDH Operator
+
+```yaml
+extraEnvs:
+  envs:
+    - name: SEGMENT_WRITE_KEY
+      value: <segment_key>
+```
+
+If you wish to subsequently disable telemetry data collection, use one of the following methods described below.
+
+### Disable Telemetry
+
+To turn off Segment telemetry, you need to deactivate the Segment provider plugin
+
+#### Using Helm Chart
+
+When you are using Helm Chart, you need to modify Helm values by adding the following configuration (if the Segment plugin is already present in your configuration, you can just change `disabled: false` to `disabled: true`):
+
+```yaml
+global:
+  dynamic:
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: true
+```
+
```

Comment:
NOTE: If the `analytics-provider-segment`  plugin is already present in your Helm configuration file, set the value of the `plugins.disabled` parameter to `true`.